code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -8,12 +8,12 @@
 	meat = /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant/zombie
 	species_traits = list(NOBREATH,RESISTCOLD,RESISTPRESSURE,NOBLOOD,RADIMMUNE,NOZOMBIE,EASYDISMEMBER,EASYLIMBATTACHMENT)
 	mutant_organs = list(/obj/item/organ/tongue/zombie)
+	
 
 /datum/species/zombie/infectious
 	name = "Infectious Zombie"
 	id = "memezombies"
 	limbs_id = "zombie"
-	no_equip = list(slot_wear_mask, slot_head)

```

Comment:
I thought mirror zombies only became the base type, not the infectious version that has spreads the disease/comes back from the dead/etc.

If that's not the case I could always change, but I'd want to see at least one zombie wizard in action.... for science (also speed = 2 on a solo antag would be suicide).